CRM/Utils/Address/BatchUpdate.php

```diff
@@ -287,7 +287,18 @@ public function processContacts($processGeocode, $parseStreetAddress) {
   public function returnResult() {
     $result = array();
     $result['is_error'] = $this->returnError;
-    $result['messages'] = implode("", $this->returnMessages);
+    $result['messages'] = '';
+    // Pad message size to allow for prefix added by CRM_Core_JobManager.
+    $messageSize = 255;
+    // Ensure that each message can fit in the civicrm_job_log.data column.
+    foreach ($this->returnMessages as $message) {
```

Comment:
I think what you want is to use CRM_Utils_String::ellipsify - e.g check elsewhere

```

$pValue = CRM_Utils_String::ellipsify($pValue, $maxLength);
```

Reply:
This is an HTML string, so we can't just truncate it without generating invalid HTML.

Reply:
ah OK - well I think it's hard to see a case where this change would cause problems so I'll merge it